File: java/src/test/java/codingdojo/MessageEnricherTest.java
package codingdojo;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import java.util.Arrays;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class MessageEnricherTest {

    @Test
    void sampleTest() {
        MessageEnricher enricher = new MessageEnricher();
        SpreadsheetWorkbook worksheet = new SpreadsheetWorkbook();
        Exception e = new RuntimeException("Terrible problem");
        ErrorResult actual = enricher.enrichError(worksheet, e);
        assertEquals("Fixme", actual.getMessage());
    }

}

File: csharp/StarterProject/MessageEnricherTest.cs
using System;
using Xunit;

namespace StarterProject
{
    public class MessageEnricherTest
    {
        [Fact]
        public void SampleTest()
        {
            var enricher = new MessageEnricher();

            var worksheet = new SpreadsheetWorkbook();

            var e = new Exception("Terrible problem");

            var actual = enricher.EnrichError(worksheet, e);

            Assert.Equal("Fixme", actual.GetMessage());
        }
    }
}